src/components/IstioCertsInfo/IstioCertsInfo.tsx

```diff
@@ -0,0 +1,181 @@
+import * as React from 'react';
+import * as API from '../../services/Api';
+import {
+  Button,
+  Card,
+  CardBody,
+  CardHeader,
+  Grid,
+  GridItem,
+  Modal,
+  Title,
+  Tooltip,
+  TooltipPosition
+} from '@patternfly/react-core';
+import { KialiAppState } from 'store/Store';
+import { istioCertsInfoSelector, lastRefreshAtSelector } from 'store/Selectors';
+import { ThunkDispatch } from 'redux-thunk';
+import { KialiAppAction } from 'actions/KialiAppAction';
+import { bindActionCreators } from 'redux';
+import { IstioCertsInfoActions } from 'actions/IstioCertsInfoActions';
+import { connect } from 'react-redux';
+import { TimeInMilliseconds } from 'types/Common';
+import { CertsInfo } from 'types/CertsInfo';
+import { PFColors } from 'components/Pf/PfColors';
+import { KialiIcon } from 'config/KialiIcon';
+import { infoStyle } from 'styles/DropdownStyles';
+
+type IstioCertsInfoState = {
+  showModal: boolean;
+  certsError: Boolean;
+};
+
+type IstioCertsInfoProps = {
```

Comment:
We usually follow one convention for Redux properties. We split the strict native properties from the component from the Redux one. We do this for better readability. I find it pretty useful for troubleshooting and understand what's part of this component and what's from redux (and therefore used beyond the component).

See the example here:
https://github.com/kiali/kiali-ui/blob/ff87e9a66894ef0f13c01a7b8f086c7a282a9aa4/src/components/IstioActions/IstioActionsButtons.tsx#L10-L24

Reply:
Note that this convention is not followed 100% as we rely as well in comments (not new).
![image](https://user-images.githubusercontent.com/1662329/130639540-2750ccbf-ed89-4674-a558-2dea0eacd8c1.png)

Reply:
Actually, this type only contains Redux props. The component props are split and declared in the previous type `IstioMetricsProps`. So the main difference between my example and yours is the way you intersect both types. However, this last example makes it a bit harder to understand because devs need the comment to make that explicit. If devs see the whole type definition, then they understand that those props are only from Redux.

Reply:
I'm OK to introduce this change, it's added now. Thanks!.